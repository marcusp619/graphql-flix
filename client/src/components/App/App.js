import React from 'react';
import Header from '../Header';
import BackgroundVideo from '../BackgroundVideo';
import MovieList from '../MovieList';
import styled, { createGlobalStyle } from 'styled-components';
import './App.css';

const App = () => {
  return (
    <div className="App">
      <GlobalStyles />
      <Header />
      <BackgroundVideo />
      <TrendingTitle>Trending Now</TrendingTitle>
      <MovieList />
    </div>
  );
};

const GlobalStyles = createGlobalStyle`
    body {
        @import url('https://fonts.googleapis.com/css?family=Kanit|Roboto');
        font-family: 'Kanit', sans-serif;
    }
`;

const TrendingTitle = styled.h1`
  color: white;
  padding: 0 1rem;
  text-align: left;
`;

export default App;
