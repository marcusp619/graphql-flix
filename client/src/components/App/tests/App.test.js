import React from 'react';
import ReactDOM from 'react-dom';
import {mount, shallow} from 'enzyme'
import App from '../App';

describe('App', () => {
  it('works', () => {
    const wrapper = shallow(<App />)
    expect(wrapper).toMatchSnapshot()
  });  
})
