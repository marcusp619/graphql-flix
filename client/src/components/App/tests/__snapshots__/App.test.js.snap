// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App works 1`] = `
<ContextConsumer>
  <Component />
</ContextConsumer>
`;
